src/index.html

```diff
@@ -7,7 +7,338 @@
+              placeholder="email"
+              required
+            >
+          </label>
+          <button class="button consultation__button">
+            Consultation
+          </button>
+        </form>
+      </div>
+    </section>
+
+    <section class="section">
+      <h2 class="heading section__heading">
+        <a name="testimonials"></a>
+        Testimonials
+        <span class="heading__crown"></span>
+      </h2>
+      <div class="testimonials">
+        <div class="person">
+          <img
+            class="person__photo"
+            src="./images/people/1.jpg"
+            alt="Tim Cook"
+          >
+          <h3 class="person__name">
+            Tim Cook
+          </h3>
+          <div class="person__position">
+            CEO
+          </div>
+          <p class="person__text">
+            Nice work, Certificates National Association of Realtors (USA).
+          </p>
+        </div>
+
+        <div class="person">
+          <img
+            class="person__photo"
+            src="./images/people/2.jpg"
+            alt="Katherine Adams"
+          >
+          <h3 class="person__name">
+            Katherine Adams
+          </h3>
+          <div class="person__position">
+            Senior Vice President
+          </div>
+          <p class="person__text">
+            My soul is illuminated by an unearthly joy, as these beautiful spring morning, which I enjoy with all my heart. I’m all alone and blissfully happy in the local edge.
+          </p>
+        </div>
+
+        <div class="person">
+          <img
+            class="person__photo"
+            src="./images/people/3.jpg"
+            alt="Jonathan Ive"
+          >
+          <h3 class="person__name">
+            Jonathan Ive
+          </h3>
+          <div class="person__position">
+            Chief Design Officer
+          </div>
+          <p class="person__text">
+            When my lovely valley of steam rises and half-day sun is above an impermeable.
+          </p>
+        </div>
+      </div>
+    </section>
+
+    <section class="section">
+      <h2 class="heading section__heading">
+        <a name="contacts"></a>
+        Contact us
+        <span class="heading__crown"></span>
+      </h2>
+      <div class="contact">
+        <div class="contact__text">
+          <p>
+            <a href="mailto:hello@miami.com" class="contact__link">
+              Hello@miami.com
+            </a>
+            <br>
+            <a href="tel:+1 234 555-55-55" class="contact__link">
+              +1 234 555-55-55
+            </a>
+          </p>
+          <p>
+            400 first ave. n.<br>
+            suite 700<br>
+            Minneapolis, MN 55401
+          </p>
+        </div>
+
+        <form class="form contact__form">
+          <label class="form__label" for="contact__name">
+            <input
+              class="input form__input"
+              type="text"
+              name="contact__name"
```

Comment:
```suggestion
              name="contactName"
```
name это не элемент бэм блока, а идентификатор формы для сервера. Можно писать просто в camelCase либо kebab-case